package.json

```diff
@@ -35,5 +35,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint-plugin-react-hooks": "^2.5.0"
```

Comment:
i have a local install of prettier which did a lot of code formatting, forgot to commit it to repo though. its very nice and easy to use  and helps readability + avoiding sloppy code presentation. you should never be manually formatting JS imo